jsx/Card.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * This file contains React component for Card
+ *
+ * @author Zaliqa Rosli
+ * @version 1.0.0
+ *
+ */
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Panel from 'jsx/Panel';
+
+/**
+ * Card component - Wraps children in a customizable card
+ *
+ */
+class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    this.props.onClick(e);
+  }
+
+  render() {
+    const cursorStyle = this.props.onClick ? {
+      cursor: 'pointer',
+    } : null;
+    return (
+      <div style={{marginLeft: '5px', marginRight: '5px'}}>
```

Comment:
Hey @zaliqarosli, I believe the best practice is to have the style object for marginLeft & marginRight defined outside the div as a constant.